Pull Request, which will contain all changes from ODF 1.3  CSD03 to ODF 1.3 CS02

svanteschubert opened the pull request:
Started with a single issue and will add fixes incrementally. Flagging this PullRequest therefore as a draft, until it is ready!
**NOTE**: Using a label as the "draft pull request feature" exists only for GitHub Team & Enterprise ;-)

svanteschubert commented:
Please note, I have fixed first the ODT, by using JEdit with archive PlugIn to access, edit and save the content.xml file within the ZIP.
Afterwards, I have been using an unzip batch file to unpack the ODT into the folder.
Therefore, if the change of the unzipped content.xml is correct, so is the ODT :-)

svanteschubert commented:
At this stage, decided to better stick to the TC-agreed wording from the former issue: 
https://issues.oasis-open.org/browse/OFFICE-3873
Instead of following the wording recently suggested by Patrick:
https://lists.oasis-open.org/archives/office/202010/msg00016.html

svanteschubert commented:
Changes made so far:

1. Updated the wording according to https://issues.oasis-open.org/browse/OFFICE-3873
2. Updated IDs of deliverables references in the ODT (exchanged 'CSD03' to 'CS02' and 'CS01' to 'CSD03') 
3. Fixing in part 3 the 'broken' link from 'http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.49.6093' to 'https://ctan.mirror.norbert-ruehl.de/biblio/bibtex/base/btxdoc.pdf'
4. Fixing missing whitespace after MathML entities in HTML
5. Generated new HTML & PDF documents

mistmist reviewed (commented):
the .owl files need fixing

svanteschubert commented on src/main/resources/odf1.3/OpenDocument-v1.3-metadata.owl:
```diff
@@ -1,62 +1,59 @@
-        OWL Schema Metadata Manifest Description
-        OpenDocument-v1.3-metadata.owl
+        OWL Package Metadata Manifest Description
+        OpenDocument-v1.3-package-metadata.owl
```
Thank you, the complete file seemed wrong due to a copy/paste error of mine! :-)

svanteschubert commented on docs/odf1.3/cs02/schemas/OpenDocument-v1.3-metadata.owl:
```diff
@@ -1,62 +1,59 @@
-        OWL Schema Metadata Manifest Description
-        OpenDocument-v1.3-metadata.owl
+        OWL Package Metadata Manifest Description
+        OpenDocument-v1.3-package-metadata.owl
```
Thank you, the complete file seemed wrong due to a copy/paste error of mine! :-)

mistmist commented:
dont merge yet, the files still contain "Draft 03"...

svanteschubert commented:
The final summary of all (non-material) changes: 

* Updated the wording according to https://issues.oasis-open.org/browse/OFFICE-3873
* Updated IDs of deliverables references 
    * in all file names and added a schema directory for RNG, OWL files
    * in the ODT (exchanged 'Draft 03' to '02', 'CSD03' to 'CS02' and 'CS01' to 'CSD03') 
    * in the RNG/RNG-HTML 
    * in the OWL files 
* Fixing in part 3 the 'broken' link from 'http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.49.6093' to 'https://ctan.mirror.norbert-ruehl.de/biblio/bibtex/base/btxdoc.pdf'
* Fixing ODF 1.2 XML manually by removing ODF 1.3 attributes to become valid ODF 1.2
* Fixing missing whitespace after MathML entities in HTML
* Generated new HTML & PDF documents
